appveyor.yml

```diff
@@ -78,7 +82,7 @@ build_script:
     if ($isBuildingPR) {
         git config user.email "gitextensions@github.com"
         git config user.name "Git Extensions"
-        git -c core.autocrlf=false commit -a -m "restore merge commit" | Out-Null
+        git -c core.autocrlf=false commit -a -m "commit version files"
         # ignore warning about line-endings conversion
         $global:LASTEXITCODE = 0
```

Comment:
This just occurred to me - we're doing all of these gimmicks because we want to set the PR commit's hash, right? Why not set override the hash explicitly instead of resetting and recommitting?